### ISSUE TITLE
multi-line properties incorrectly show errors

### ISSUE THREAD

**Issue by kaitlynbrown** [Microsoft/typescript-styled-plugin#44]
I have a fairly long css property that `prettier` has decided should be spread onto multiple lines:
![image](https://user-images.githubusercontent.com/1388240/37994212-b61c3ba2-31c5-11e8-937a-3fc9b33968a3.png)
This compiles and works totally fine, but `ts-styled-plugin` seems to mistakenly think there are errors, as you can see by the red squiggly lines in the screenshot. The first "error" is `semi-colon expected` and the second is `{ expected`

**Comment by mjbvz** [comment]
Can you please share the source code of that example so I can make sure it is properly fixed

**Comment by kaitlynbrown** [comment]
```
import styled from 'styled-components';

export const SidebarLink = styled.a`
  display: flex;
  width: 6rem;
  height: 5rem;
  margin-right: -3px;
  border-right: 3px solid
    ${({ active, theme: { colors } }) =>
      active ? colors.yellow : "transparent"};
  border-bottom: 1px solid rgba(255, 255, 255, 0.5);
  font-weight: bold;
  font-size: 0.875rem;
  color: white;
  cursor: pointer;
  &:not([href]):not([tabindex]) {
    color: white;
  }
  &:hover {
    color: inherit;
    text-decoration: none;
  }
`;
```

**Comment by mjbvz** [comment]
Thanks. Should be fixed in 0.6.2
